mlflow/store/artifact/unity_catalog_models_artifact_repo.py

```diff
@@ -120,6 +124,11 @@ def _get_artifact_repo(self, lineage_header_info=None):
         Get underlying ArtifactRepository instance for model version blob
         storage
         """
+        if MLFLOW_USE_DATABRICKS_SDK_MODEL_ARTIFACTS_REPO_FOR_UC.get():
+            return DatabricksSDKModelsArtifactRepository(self.model_name, self.model_version)
+        else:
+            from mlflow.utils.logging_utils import eprint
+            eprint("Not using DatabricksSDKModelsArtifactRepository")
```

Comment:
`eprint` should be used for an error or a warning. Is this an error or a warning?

Reply:
I am using these eprint for debugging, will remove them before merging